common/src/main/resources/assets/wynntils/lang/en_us.json

```
@@ -184,19 +186,13 @@
   "feature.wynntils.contentTrackerOverlay.overlay.contentTracker.title": "Tracked",
   "feature.wynntils.cosmeticsPreview.description": "Renders your player in the cosmetics screen.",
   "feature.wynntils.cosmeticsPreview.name": "Cosmetics Player Preview",
-  "feature.wynntils.customBankPages.buttonFivePage.description": "Which page should button 5 jump to?",
-  "feature.wynntils.customBankPages.buttonFivePage.name": "Button 5 page",
-  "feature.wynntils.customBankPages.buttonFourPage.description": "Which page should button 4 jump to?",
-  "feature.wynntils.customBankPages.buttonFourPage.name": "Button 4 page",
-  "feature.wynntils.customBankPages.buttonOnePage.description": "Which page should button 1 jump to?",
-  "feature.wynntils.customBankPages.buttonOnePage.name": "Button 1 page",
-  "feature.wynntils.customBankPages.buttonSixPage.description": "Which page should button 6 jump to?",
-  "feature.wynntils.customBankPages.buttonSixPage.name": "Button 6 page",
-  "feature.wynntils.customBankPages.buttonThreePage.description": "Which page should button 3 jump to?",
-  "feature.wynntils.customBankPages.buttonThreePage.name": "Button 3 page",
-  "feature.wynntils.customBankPages.buttonTwoPage.description": "Which page should button 2 jump to?",
-  "feature.wynntils.customBankPages.buttonTwoPage.name": "Button 2 page",
-  "feature.wynntils.customBankPages.description": "Enable custom quick jump buttons in the bank?",
+  "feature.wynntils.customBankPages.bankDestinations.description": "Which pages would you like your bank quick jump buttons to jump to? Pages should be separated with an single comma.",
```

```suggestion
  "feature.wynntils.customBankPages.bankDestinations.description": "Which pages would you like your bank quick jump buttons to jump to? Enter a list of six numbers separated with a comma.",
```

(and similar for the others)